File: java/Somatorio.java
public class Somatorio {

    public static int somatorio(int[] numeros) {
        int soma = 0;
        for (int i = 0; i < numeros.length; i++) {
            soma += numeros[i];
        }
        return soma;
    }

    public static void main(String[] args) {
        System.out.println("--- Somatório ---");
        System.out.println("Somatório de [1,2,3,4,5] = " + somatorio(new int[]{1, 2, 3, 4, 5}));
    }
}

File: javascript/somatorio.js
function somatorio(numeros) {
  let soma = 0;
  for (let i = 0; i < numeros.length; i++) {
    soma += numeros[i];
  }
  return soma;
}

console.log("--- Somatório ---");
console.log(`Somatório de [1,2,3,4,5] = ${somatorio([1, 2, 3, 4, 5])}`);
